src/Microsoft.Health.Fhir.Tests.Common/Microsoft.Health.Fhir.Tests.Common.csproj

```diff
@@ -217,6 +220,8 @@
     <None Remove="TestFiles\Stu3\PatientWithMinimalData.json" />
     <None Remove="TestFiles\Stu3\ProvenanceHeader.json" />
     <None Remove="TestFiles\Stu3\Sequence.json" />
+    <None Remove="TestFiles\Stu3\Profile-Patient-PassUsCore-Example.json" />
+    <None Remove="TestFiles\Stu3\Profile-Patient-FailUsCore-Example.json" />
```

Comment:
These need to be added as EmbeddedResources like the files in Normative.

Reply:
Done.